src/components/Spinner/Spinner.styles.js

```diff
@@ -0,0 +1,28 @@
+import styled, { keyframes } from 'styled-components';
+import { ReactComponent as SpinnerSVG } from './loading-spinner.svg';
+
+const rotate = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+export const SpinnerWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-self: center;
+  animation: ${rotate} 1.5s linear infinite;
+  padding: 2rem 1rem;
+  width: 71px;
```

Comment:
Seems to work without width and height. They are set inside the loading-spinner.svg, probably that's why